ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -200,6 +203,14 @@ private void updateLayout(View viewToUpdate, int x, int y, int width, int height
     }
   }
 
+  /**
+   * Creates a {@link View} and adds it to a corresponding {@link ViewManager}.
```

Comment:
nit: Doesn't always create a `{@link View}`, it could be a ScrollView for example.